design/features/routing/README.md

```diff
@@ -1,40 +1,78 @@
-# Simple routing logic (skip questions)
+# Routing 
 
-## Status
+## Status 
 
-Date created: *2023-05-23*  
-
-Epic trello card: https://trello.com/c/nBnFZQlw/135-epic-simple-routing  
-Mural working board: https://app.mural.co/invitation/mural/gaap0347/1671196865223?sender=ue1ef9fc3c2ad3697c3c93132&key=09616cf9-ae28-43a7-8810-10970a59a765  
-Basic routing feature write up: https://docs.google.com/document/d/1i7T5D3PQw4hB1wgISi-r5086Fzw1-3nKnwaCGbNMi1E/edit#heading=h.h2rnqdwxl250  
-___
+Date updated: *2024-10-31*  
 
 ## Contents
 
+- [Branching and exit pages](#branching-and-exit-pages-iteration-1)
+  - [What](#what)
+  - [Why](#why)
+  - [Hypothesis](#hypothesis)
+
 - [Simple routing logic (skip questions)](#simple-routing-logic-skip-questions)
-  - [Status](#status)
-  - [Contents](#contents)
   - [What](#what)
   - [Why](#why)
   - [Hypothesis](#hypothesis)
 
 ___
 
-<br>
+## Branching and exit pages (iteration 1)
+
+Date updated: *2024-10-24* 
+
+Epic trello card: https://trello.com/c/a5c3JaWu/1908-epic-branching-and-exit-pages  
+Mural working board: https://app.mural.co/t/gaap0347/m/gaap0347/1728478914347/6a7e0b709f50f00f81cab37414fdf44ec8601a70?wid=0-1729257264973  
+Routing and branching terminology (how we decided what to deliver): https://docs.google.com/document/d/1lXKKGyT6kbFc0EbthTPiEHK5sEIc1EuVqSLXuQ1MXPc/edit?usp=sharing  
+
+### What  
+
+This feature introduces new functionality to the form building tool to help form creators create simple skip logic for questions. This means you can skip someone to a later question in a form based on their response to a question where they have to select one answer from a list.
+
+#### User stories
+
+**As a form creator**, I need to allow users to skip to the relevant group of questions based on their previous answer so that we only collect the information needed.  
+
+**As a form filler**, I need to skip one or more questions based on my previous answer so that I don’t have to give information that is not relevant to me.  
+
+**As a form processor**, I need to receive the information when people have taken different branches in a way that I can easily add it up.  
```

Comment:
Noticed that these user stories seem to differ from those in the ['User stories - branching and exit pages' doc](https://docs.google.com/document/d/1hnKX2jUxlGXnGhPeIZ9ByTB4bNZ6-OioGKn2LvIESY4/edit?tab=t.0), so wondered which we should be including? 

The form processor story in particular is a bit unclear to me - not sure what "[...] in a way that I can easily add it up" means?

Reply:
Yep, may need to double check these and tidy them up so they're consistent. I can look at this next week if you like.